shared/js/debug.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * This exposes some modules we use for testing via the background page console.
+ * NOTE this is not added to the release version of the extension
+ */
+const settings = require('./settings')
+const abp = require('abp-filter-parser')
+
+window.dbg = {
+  settings,
+  abp
+}
```

Comment:
@jdorweiler I exposed them as `dbg.settings` and `dbg.abp` instead of `bkg` because it's slightly more descriptive than.